File: app/src/main/java/com/rdev/tryp/model/sign_up_response/SignUpResponse.java
package com.rdev.tryp.model.sign_up_response;

import com.google.gson.annotations.SerializedName;

public class SignUpResponse {

    @SerializedName("data")
    private Data data;

    public void setData(Data data) {
        this.data = data;
    }

    public Data getData() {
        return data;
    }

    @Override
    public String toString() {
        return
                "SignUpResponse{" +
                        "data = '" + data + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/rdev/tryp/model/sign_up_response/SignUpResponse.kt
package com.rdev.tryp.model.sign_up_response

import com.google.gson.annotations.SerializedName


class SignUpResponse {

    @SerializedName("data")
    var data: Data? = null

    override fun toString(): String {
        return "SignUpResponse{" +
                "data = '" + data + '\''.toString() +
                "}"
    }

}